tools/validators/instance_validator/tests/entity_instance_test.py

```diff
@@ -201,6 +201,20 @@ def testValidateMultipleCompliantTranslationWithNamespaceOther(self):
     if not instance_lighting.IsValidEntityInstance():
       self.fail('exception incorrectly raised')
 
+  def testValidateMultipleCompliantTranslationWithIdenticalTypes(self):
+    parsed = instance_parser.parse_yaml(os.path.join(_TESTCASE_PATH,
+                                                       'GOOD',
+                                                       'good_translation'
+                                                       '_identical.yaml'))
```

Comment:
(non-blocking) When we redo this, we should separate parsing of the different syntaxes and the validation of the contents by creating a native object as a go-between.

Reply:
agreed